Exercise1/exc1/divider/divider/scripts/divider.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import Float64
+q = 0.15
```

Comment:
```suggestion
from std_msgs.msg import Float64


q = 0.15
```

You would normally leave two lines between the imports and your first statement, this is part of the PEP-8 guidelines that we use. Please try to take a look at them :slightly_smiling_face: